### ISSUE TITLE
Importing into database after 13 hours still nothing?!

### ISSUE THREAD

**Issue by BobWs** [firefly-iii/firefly-iii#2747]
Hello,
I try to migrate my MoneyWiz accounts data to Firefly but after almost 13 hours of importing, little has happened. There are only some of the Expense accounts what has imported. But further no other data has been imported. My account data runs from ca. 2004 till now. Is there a limit to what you can import into Firefly?

**Comment by JC5** [comment]
13 hours? And you just let it run?

There is no limit. But it might be smarter not to do this in a big bang but split up your import into smaller files. This will allow you to correct mistakes for example. You can track the progress using the log files.

**Comment by BobWs** [comment]
> 13 hours? And you just let it run?
> 
> There is no limit. But it might be smarter not to do this in a big bang but split up your import into smaller files. This will allow you to correct mistakes for example. You can track the progress using the log files.

Yes I thought let it run and will see what happens;-)
Okay I will split it up and see what happens. How small batches do you recommend? Is a lot of data!

**Comment by JC5** [comment]
Try a 1000 rows, and I'll send you some hints to check the logs.

**Comment by JC5** [comment]
Hi! This is a canned reply that tells you how to get more debug details.

1. Please collect some log files. Go to `storage/logs` and open the latest log file. It should be called something like "ff3-fpm-fcgi-*.log". In it, you should be able to find the error along with its "stacktrace": this will tell me where Firefly III broke down. If you run Docker, use `docker logs <container>`.
2. You may also [enable debug mode](https://github.com/firefly-iii/help/wiki/Enable-debug-logging-and-debug-mode) to make sure the log files catch everything and try again.
3. Make sure you clear your browser cookies and clear Firefly III's cache files by running `php artisan cache:clear` or by visiting `/flush` on your installation.

If you do not want to share log files or other details on GitHub, please [send them to me over (encrypted) email](https://firefly-iii.org/contact-information.html).

**Comment by BobWs** [comment]
Looks like the import is working I see a lot of this in the logfile:
`[2019-10-19 12:56:35] local.INFO: Linked 9509 journals to tag #5 ("Import with code 'hW3pxDeVprfP'") ` 
`[2019-10-19 12:56:35] local.INFO: Linked 9509 journals to tag #5 ("Import with code 'hW3pxDeVprfP'") ` 
`[2019-10-19 12:56:35] local.INFO: Linked 9509 journals to tag #5 ("Import with code 'hW3pxDeVprfP'")  `
`[2019-10-19 12:56:35] local.INFO: Linked 9509 journals to tag #5 ("Import with code 'hW3pxDeVprfP'")`

I also saw this in the logfile:
`[2019-10-19 09:59:44] local.ERROR: Could not decrypt data of attachment #11: The payload is invalid. ` 
`[2019-10-19 09:59:44] local.ERROR: Could not decrypt data of attachment #12: The payload is invalid. ` 
`[2019-10-19 09:59:44] local.ERROR: Could not decrypt data of attachment #12: The payload is invalid.  `
`[2019-10-19 09:59:44] local.ERROR: Could not decrypt data of attachment #12: The payload is invalid.  `
`[2019-10-19 10:00:32] local.ERROR: Could not decrypt data of attachment #11: The payload is invalid.`

But I don't have any attachments in Firefly. 
Do you have any idea what it could be?

**Comment by JC5** [comment]
Both aren't a problem, but the "linked" message should have appeared only once.

**Comment by BobWs** [comment]
Everything imported okay! Database running smooth.
Thanks for you're patients and great software!

**Comment by JC5** [comment]
No problem, glad you like it :+1:
